app/Models/Models/CovidCaseModel.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Models\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+
+class CovidCaseModel extends Model
+{
+    use HasFactory;
+
+    protected $table = 'covid_case';
```

Comment:
Model name and table, pls follow laravel conventions:
- table name with plural form
- You don't need to have a `Model` suffix.
- Pls change all models